test/HiGlassComponentTest.js

```diff
@@ -3133,4 +3133,21 @@ describe('Simple HiGlassComponent', () => {
     });
 
   });
+
+  
+  describe('Cleanup', () => {
```

Comment:
Maybe use an `after` instead, since this isn't really a test?

Reply:
Leaving this one be because that test file is a huge can of worms that I don't want to disturb.